packages/web-components/src/components/button/button.ts

```diff
@@ -246,10 +149,5 @@ class DDSButtonExpressive extends FocusMixin(StableSelectorMixin(LitElement)) {
   static styles = styles; // `styles` here is a `CSSResult` generated by custom WebPack loader
 }
 
-console.warn(
-  'The dds-button-expressive component has been deprecated in favor of the core carbon button component using the isExpressive prop. ' +
-    'See carbon core button documentation for more information.'
-);
-
 /* @__GENERATE_REACT_CUSTOM_ELEMENT_TYPE__ */
-export default DDSButtonExpressive;
```

Comment:
looking good! my only concern is that we were planning to deprecate `DDSButtonExpresive` but this would be a breaking change. I don't think many people use `DDSButtonExpressive` on it's own, but might want to check with the team before merging

Reply:
You're right, I added back the `DDSButtonExpressive` component along with a deprecation notice 👍